src/Component/Nav/Nav.js

```diff
@@ -1 +1,23 @@
 import React, { Component } from 'react';
+import './Nav.scss';
+
+class Nav extends Component {
+  render() {
+    return (
+      <nav className="navBar">
+        <div className="navWrap">
+          <div className="navContainer">
+            <img className="logo" src="/images/logo.jpg" alt="로고이미지" />
+            <div className="navRightBar">
+              <button className="navRightLanguageBtn">한국어</button>
+              <button className="navRightWonBtn">KRW 대한민국 원(₩)</button>
+              <button className="navLogin">로그인</button>
+            </div>
+          </div>
+        </div>
+      </nav>
```

Comment:
```suggestion
      <nav className="navBar">
        <img className="logo" src="/images/logo.jpg" alt="로고이미지" />
        <div className="navRightBar">
          <button className="navRightLanguageBtn">한국어</button>
          <button className="navRightWonBtn">KRW 대한민국 원(₩)</button>
          <button className="navLogin">로그인</button>
        </div>
      </nav>
```

navWrap와 navContainer 이 두 영역은 불필요하네요 :)